test/cra-template/README.md

```diff
@@ -0,0 +1,20 @@
+# cra-template-typescript
+
+This is the official TypeScript template for [Create React App](https://github.com/facebook/create-react-app).
+
+To use this template, add `--template typescript` when creating a new app.
+
+For example:
+
+```sh
+npx create-react-app my-app --template typescript
+
+# or
+
+yarn create react-app my-app --template typescript
```

Comment:
is it possible to start a JS template too avoiding this option? and if yes, can we say that explicitly here?

Reply:
DONE